--- origsrc/pycdio-0.13/configure.ac	2007-10-21 17:28:58.000000000 -0500
+++ src/pycdio-0.13/configure.ac	2008-04-24 14:28:38.937500000 -0500
@@ -118,7 +118,7 @@
   AC_CANONICAL_HOST
   case $host_os in
     cygwin* | mingw* | pw32*)
-      PYTHON_LIBS=`$PYTHON ./get-cygwin-python-libs`
+      PYTHON_LIBS=`$PYTHON ${srcdir}/get-cygwin-python-libs`
       SO='.dll'
       CFLAGS="$CFLAGS -DUSE_DL_IMPORT"
 
